### PR TITLE
Updated docs for store installation

### PR DESCRIPTION
**Pull request by scooley**
Installation instructions for the Linux Subsystem on Windows 10

### CODE REVIEW

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[commandline/WSL/install_guide.md](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/install_guide.md) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuInstall.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuInstall.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuStore.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuStore.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5C44a4c0e9-f7fc-ae3e-a431-dcb870cfc899%5CPullRequest%5C201707120021162221-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[commandline/WSL/install_guide.md](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/install_guide.md) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuInstall.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuInstall.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuStore.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuStore.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5C44a4c0e9-f7fc-ae3e-a431-dcb870cfc899%5CPullRequest%5C201707120025251828-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: commandline/WSL/install_guide.md [hunk 1]
```diff
@@ -1,84 +1,93 @@
...
-1. From Start, search for "Turn Windows features on or off" (type 'turn')
-2. Select **Windows Subsystem for Linux (beta)**  
-  ![](media/windowsFeatures.png)
-  
-3. Click OK
+1. Open the Windows Store and choose your favorite Linux distribution.      
+    Here are links directly to the store installers:
+    * [Ubuntu](https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6)
+    * [OpenSUSE](https://www.microsoft.com/store/apps/9njvjts82tjx)
+    * [SLES](https://www.microsoft.com/store/apps/9p32mwbh6cns)
```
Comment (sunilmut) [inline commandline/WSL/install_guide.md]: OpenSUSE and SLES are still not available at the store. It's likely that this doc will get published before they go online. Should we hold off to publishing these links until they go online?

File: commandline/WSL/install_guide.md [hunk 2]
```diff
@@ -1,84 +1,93 @@
...
-In order to run Bash on Windows, you will need to manually:
-1. Turn-on Developer Mode
-2. Enable the "Windows Subsystem for Linux (beta)" feature via the GUI or the command-line:
+## For Windows Insiders -- Install Linux distribution of choice
 
-### Turn on Developer Mode
-1. Open Settings -> Update and Security -> For developers
-2. Select the Developer Mode radio button  
-  ![](media/updateAndSecurity.png)
+> This section is for Windows Insiders (build 16215 or later).  For earlier versions of Windows 10, follow [these instructions using lxrun](install_guide.md#Install-using-lxrun-Anniversary-Update-and-Creators-Update).
```
Comment (sunilmut) [inline commandline/WSL/install_guide.md]: "build 16215 or later" -> Often times our customers don't know how to find out their build number. Can we add a note for the user to find out their build number to reduce ambiguity? Type `ver` in cmd prompt or search for `msinfo32` and look at version are couple of ways we have used to establish a baseline here.

File: commandline/WSL/install_guide.md [hunk 3]
```diff
@@ -1,84 +1,93 @@
...
-Open a PowerShell prompt as administrator and run:
+2. Select "Get"
 
-```PowerShell
-Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
-```
+    >  **Troubleshooting:  Installation failed with error 0x8007007e**  
+    >  This error occurs when your system doesn't support Linux from the store.  Make sure that:
+    > * You're running Windows build 16215 or later. [Check your build](install_guide.md#prerequisites).
+    > * The Windows Subsystem for Linux optional component is enabled. [Instructions here](install_guide.md#Install-the-Windows-Subsystem-for-Linux).
```
Comment (sunilmut) [inline commandline/WSL/install_guide.md]: Can we mention that installing on non-system drive is not supported and they might see the error code 0x80070003

File: commandline/WSL/install_guide.md [hunk 4]
```diff
@@ -1,84 +1,93 @@
...
 
-![](media/system.png) 
+## Install the Windows Subsystem for Linux
+Enable the "Windows Subsystem for Linux" optional feature and reboot.
 
-If your build is below 14393, try checking for updates.
+1. Open PowerShell as Administrator and run:
+    ``` PowerShell
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+    ```
```
Comment (sunilmut) [inline commandline/WSL/install_guide.md]: Alternatively, they can go to "Turn Windows feature on or off" and enable the "Windows Subsytem for Linux" feature.

File: commandline/WSL/install_guide.md [hunk 5]
```diff
@@ -1,84 +1,93 @@
 ---
-title: Bash on Ubuntu on Windows - Installation Guide
-description: Installation Guide
-keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu
-author: jackchammons
-ms.date: 9/6/2016
+title: Install Linux Subsystem on Windows 10
```
Comment (scooley) [inline commandline/WSL/install_guide.md]: Agreed.  I remember there was a big conversation around which term to use but now that the bash part and Ubuntu part are through the store, it makes sense as a systemic change.

File: commandline/WSL/install_guide.md [hunk 6]
```diff
@@ -1,84 +1,93 @@
...
-After installation your Linux distribution will be located at: `%localappdata%\lxss\`
-This directory is marked as a hidden system folder for a very good reason: 
+    Select the Developer Mode radio button  
+    ![](media/updateAndSecurity.png)
 
-> **Avoid creating and/or modifying files in this location using Windows tools 
-> and apps!** If you do, it is likely that your Linux files will be corrupted and data loss may occur. 
-> Please read this [blog post](https://blogs.msdn.microsoft.com/commandline/2016/11/17/do-not-change-linux-files-using-windows-apps-and-tools/) 
-> for more information.
+    Restart your Computer.
```
Comment (scooley) [inline commandline/WSL/install_guide.md]: Good catch -- forgot to scrub it when I pulled the optional component enabling steps out

File: commandline/WSL/install_guide.md (hunk 3 above)
Comment (scooley) [inline commandline/WSL/install_guide.md]: Done

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[commandline/WSL/install_guide.md](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/install_guide.md) | :white_check_mark:Succeeded | |
[commandline/WSL/media/AppStorage.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/AppStorage.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuInstall.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuInstall.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuStore.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuStore.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5C44a4c0e9-f7fc-ae3e-a431-dcb870cfc899%5CPullRequest%5C201707121843085173-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[commandline/WSL/install_guide.md](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/install_guide.md) | :white_check_mark:Succeeded | |
[commandline/WSL/media/AppStorage.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/AppStorage.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuInstall.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuInstall.png) | :white_check_mark:Succeeded | |
[commandline/WSL/media/UbuntuStore.png](https://github.com/MicrosoftDocs/CommandLine-Docs/blob/win10-install/commandline/WSL/media/UbuntuStore.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5C44a4c0e9-f7fc-ae3e-a431-dcb870cfc899%5CPullRequest%5C201707121849418372-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**